commons/com.b2international.index/src/com/b2international/index/es/EsDocumentSearcher.java

```diff
@@ -397,8 +396,14 @@ private void addSort(DocumentMapping mapping, SearchSourceBuilder reqSource, Sor
 					// XXX: default order for scores is *descending*
 					reqSource.sort(SortBuilders.scoreSort().order(sortOrder)); 
 					break;
-				case DocumentMapping._DOC: //$FALL-THROUGH$
-					field = DocumentMapping._DOC;
+				case "_default":
```

Comment:
Please add the fall-through marker back; it is hard to spot the absence of a `break` statement, sometimes even with the comment present 😄👓